crates/core_arch/src/arm_shared/ex.rs

```diff
@@ -5,9 +5,11 @@
 // Not supported: v5, v6-M
 // NOTE: there's no dedicated CLREX instruction in v6 (<v6k); to clear the exclusive monitor users
 // have to do a dummy STREX operation
+#[cfg(target_feature = "aarch64")]
 #[cfg(any(
     all(target_feature = "v6k", not(target_feature = "mclass")), // excludes v6-M
     all(target_feature = "v7", target_feature = "mclass"), // v7-M
+    doc
 ))]
```

Comment:
```suggestion
#[cfg(any(
    target_feature = "aarch64",
    all(target_feature = "v6k", not(target_feature = "mclass")), // excludes v6-M
    all(target_feature = "v7", target_feature = "mclass"), // v7-M
    doc
))]
```

Same with the others below.